src/index.html

```diff
@@ -1,32 +1,43 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <script type="module" src="index.js" defer></script>
-    <script type="module" src="progressBar.js" defer></script>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
```

Comment:
Keep the inset since `<html>` in a parent component to `<head>` and `<body>`.

Reply:
it's also due to my auto-formating on save, I removed it to avoid unnecessary changes in the future.